src/main/java/com/teragrep/pth10/ast/UnquotedText.java

```diff
@@ -53,7 +54,7 @@
  */
 public class UnquotedText implements Text {
```

Comment:
i think this should take the quote character as an argument instead of using regex to match it?

Reply:
Would it be okay if no quote character is provided it would remove all, and if it is provided it only removes the specific one? I think currently UnquotedText is used to remove all on purpose